### PR TITLE
Add toDateUTC function + fix font color. (PT-187039201)

### PR DESCRIPTION
**Pull request by lublagg**

### CODE REVIEW

File: src/maps/components/colorbar-plot.tsx [hunk 1]
```diff
@@ -102,11 +101,9 @@ const ColorbarPlot = () => {
         const observation = observations[date][position.index] ?? 0;
         if (observation > lastValue && observation <= bin.value) {
           const key = placename(position, placenames);
-          if (index !== ymdDates.length) {
-            const startValue = formatDate(new Date(date));
-            const endValue = formatDate(new Date(yLabels[index + 1]));
-            data.push({x: key, y: [startValue, endValue]});
-          }
+          const startValue = formatDate(toDateUTC(date));
+          const endValue = yLabels[index + 1];
```
Comment (dougmartin) [inline src/maps/components/colorbar-plot.tsx:105]: We've dropped the date formatting for `endValue` here - was that intentional?

**Review by dougmartin — COMMENTED** [review]
Maybe saw an issue with the `endValue` update.

File: src/maps/components/colorbar-plot.tsx (hunk 1 above)
Comment (lublagg) [reply src/maps/components/colorbar-plot.tsx:105]: Yes - yLabels is a different array created from the ymdDates array, and the labels are already formatted.

**Review by dougmartin — APPROVED** [review]
Looks good :+1:

**Pull request merged**